Brocfile.js

```diff
@@ -1,174 +1,28 @@
-var moveFile        = require('broccoli-file-mover');
-var wrap            = require('broccoli-wrap');
-var jshint          = require('broccoli-jshint');
-var defeatureify    = require('broccoli-defeatureify');
-var version         = require('git-repo-version')(10);
-var yuidoc          = require('broccoli-yuidoc');
-var replace         = require('broccoli-replace');
-var path            = require('path');
-
-function minify(tree, name){
-  var config = require('./config/ember-defeatureify');
-  tree = defeatureify(tree, {
-    debugStatements: config.options.debugStatements,
-    enableStripDebug: config.stripDebug
-  });
-  tree = moveFile(tree, {
-    srcFile: name + '.js',
-    destFile: '/' + name + '.prod.js'
-  });
-  var uglified = moveFile(uglify(tree, {mangle: true}),{
-    srcFile: name + '.prod.js',
-    destFile: '/' + name + '.min.js'
-  });
-  return merge([uglified, tree], {overwrite: true});
-}
-
-function testTree(packageName){
-  var test = pickFiles('packages/' + packageName + '/tests', {
-    srcDir: '/',
-    files: [ '**/*.js' ],
-    destDir: '/' + packageName
-  });
-  var jshinted = jshint('packages/' + packageName + '/', {
-    jshintrcPath: path.join(__dirname, '.jshintrc')
-  });
-  jshinted = wrap(jshinted, {
-    wrapper: [ "if (!QUnit.urlParams.nojshint) {\n", "\n}"],
-  });
-  jshinted = pickFiles(jshinted, {
-    files: ['{lib,tests}/**/*.js'],
-    srcDir: '/',
-    destDir: '/' + packageName + '-jshint'
-  });
-  return merge([jshinted, test]);
-}
-
-var yuidocTree = yuidoc('packages', {
-  srcDir: '/',
-  destDir: 'docs',
-  yuidoc: {
-    "name": "The ember-data API",
-    "description": "The ember-data API: a data persistence library for Ember.js",
-    "version": version,
-    "logo": "http://f.cl.ly/items/1A1L432s022u1O1q1V3p/ember%20logo.png",
-    "url": "https://github.com/emberjs/data",
-    "options": {
-      "paths": [
-        "packages/ember-data/lib",
-        "packages/activemodel-adapter/lib",
-        "packages/ember-inflector/lib"
-      ],
-      "exclude": "vendor",
-      "outdir":   "docs/build"
-    }
+// To create fast production builds (without ES3 support, minification, derequire, or JSHint)
+// run the following:
+//
+// DISABLE_ES3=true DISABLE_JSCS=true DISABLE_JSHINT=true DISABLE_MIN=true DISABLE_DEREQUIRE=true ember serve --environment=production
+
+var EmberBuild         = require('emberjs-build');
+var packages           = require('./lib/packages');
+var vendoredPackage    = require('emberjs-build/lib/vendored-package');
+var vendoredES6Package = require('emberjs-build/lib/es6-vendored-package');
+
+var emberBuild = new EmberBuild({
+  name: 'ember-data',
+  namespace: 'DS',
+  packages: packages,
+  skipTemplates: true,
+  skipRuntime: true,
+  vendoredPackages: {
+    'loader': vendoredPackage('loader'),
+    'ember-inflector': vendoredES6Package('ember-inflector', {
+      libPath: 'bower_components/ember-inflector/packages/ember-inflector/lib',
```

Comment:
:ok_hand: that seems to have worked, but now 
![screen shot 2015-01-26 at 5 20 45 pm](https://cloud.githubusercontent.com/assets/2575027/5909551/b55f3884-a57f-11e4-8379-ffd8b2453374.png)

I can't find `qunit_configuration` and 0 tests are running.

I might just hold off until we have our hangout. I don't think  I fully understand all the moving parts